ui/perfherder/alerts/StatusDropdown.jsx

```diff
@@ -44,9 +45,98 @@ export default class StatusDropdown extends React.Component {
       showNotesModal: false,
       showTagsModal: false,
       selectedValue: this.props.issueTrackers[0].text,
+      jobList: [],
     };
   }
 
+  containsVismet = (title) => {
+    const vismetMetrics = [
```

Comment:
We should rather move this to the constants module, so we don't initialize this over & over again.

Reply:
The new change isn't good enough. We still initialize the `vismetMetrics` array for each call to `containsVismet`. And there can be as many calls as there are alerts in a summary.

Reply:
should I move just `vismetMetrics` to the constants and keep the function with the logic in `StatusDropdown.jsx` and import just `vismetMetrics` from contants?